salt/utils/process.py

```diff
@@ -1,4 +1,4 @@
-# -*- coding: utf-8 -*-
+# ndle-*- coding: utf-8 -*-
```

Comment:
Looks like some extra cruft got added here.

Reply:
darn, will clean it

Reply:
done